samples/DotNet/Azure.Messaging.EventHubs/AnomalyDetector/Producer/Program.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Producer
+{
+    using System;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Azure.Messaging.EventHubs;
+    using System.Collections.Generic;
+    using System.IO;
+
+    public class Program
+    {        
+        private const string EventHubConnectionString = "<EVENT HUBS NAMESPACE CONNECTION STRING>";
```

Comment:
nit:  You may want to consider calling this `EventHubsConnectionString` to denote that it is allowing a namespace-level connection string and not requiring one specific to an individual Event Hub instance.

Reply:
Done. Renamed it to: EventHubNamespaceConnectionString